playbooks/common/openshift-etcd/config.yml

```diff
@@ -0,0 +1,106 @@
+---
+- name: Gather and set facts for etcd hosts
+  hosts: oo_etcd_hosts_to_config
+  roles:
+  - openshift_facts
+  tasks:
+  - openshift_facts:
+      role: common
+      local_facts:
+        hostname: "{{ openshift_hostname | default(None) }}"
+  - name: Check for etcd certificates
+    stat:
+      path: "{{ item }}"
+    with_items:
+    - "/etc/etcd/ca.crt"
```

Comment:
Yeah, the etcd package comes from Fedora/RHEL and creates /etc/etcd and the service uses /etc/etcd/etcd.conf for it's environment variables.

We could store the certs elsewhere, but this seemed like the right place to me.